crates/router/src/connector/forte/transformers.rs

```diff
@@ -252,14 +252,16 @@ impl<F, T>
         Ok(Self {
             status: enums::AttemptStatus::foreign_from((response_code, action)),
             response: Ok(types::PaymentsResponseData::TransactionResponse {
-                resource_id: types::ResponseId::ConnectorTransactionId(transaction_id.to_string()),
+                resource_id: types::ResponseId::ConnectorTransactionId(
+                    transaction_id.clone().to_string(),
```

Comment:
In Rust, the `to_string()` method is used to convert a value into a String type. When you call `to_string()` on a value, it creates a new String instance and copies the content of the value into that new String. This means that you get a new String that is a clone of the original value.
So clone is not required here